README.md

```diff
@@ -115,7 +171,7 @@ where the `context` directory is set to the root of the cla_public directory.
 
 :tada: :shipit:
 
-#### Kubernetes Deploy
+#### Deploy to Kubernetes using Circle CI
```

Comment:
_nitpick_ I think the service name is written as "CircleCI" (one word): https://circleci.com/docs/